react-frontend/src/components/Nav/covidbanner.js

```diff
@@ -0,0 +1,64 @@
+import React, { useState, useEffect } from 'react';
+import Alert from 'react-bootstrap/Alert';
+import {
+  EmptyBannerStyle,
+  CovidBannerStyle,
+  CovidButtonStyle,
+} from '../StyleComponents/pageContent';
+import '../../css/nav.css';
```

Comment:
Does this css do anything for us?  If it does is it possible to remove and refactor with styled components, or will that be a huge amount of work?